README.md

```diff
@@ -23,6 +23,7 @@ Each sub-directory (one per complex) contains the following files:
 * `XXX_l_u.pdb` : Unbound ligand PDB
 * `XXX_r_b-matched.pdb` : Matched bound receptor PDB
 * `XXX_l_b-matched.pdb` : Matched bound ligang PDB
+* `XXX_r_u_cg.pdb` : Coarse-grain models for the receptor proteins
```

Comment:
Shouldn't there be also a CG model for the ligand?

And may-be change "Coarse-grain" to "Martini coarse-grain"

Reply:
done

Reply:
yes, there are also for ligands